### PR TITLE
DEV-1204 - add function for adding mutliple admins

### PR DESCRIPTION
**Pull request by grp06**
## Describe your changes
- Also threw in a bunch of natspec documentation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have updated the documentation accordingly. (if applicable)
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/Raft.sol [hunk 1]
```diff
@@ -78,47 +89,86 @@ contract Raft is
...
+  ) public virtual {
+    require(_exists(tokenId), "setMultipleAdmins: tokenId does not exist");
+    require(ownerOf(tokenId) == msg.sender, "setMultipleAdmins: unauthorized");
+    require(
+      admins.length == isActive.length,
+      "setMultipleAdmins: admins and isActive arrays must have the same length"
+    );
+
+    for (uint256 i = 0; i < admins.length; i++) {
+      _admins[tokenId][admins[i]] = isActive[i];
```
Comment (stefan-girlich) [inline src/Raft.sol:148]: we safe all addresses in `_admins[tokenId]`, including those of non-admin users. does this come with any sort of penalty in terms of memory usage or cost?

**Review by stefan-girlich — APPROVED** [review]
lgtm, only some conceptual questions

File: src/Raft.sol [hunk 2]
```diff
@@ -78,47 +89,86 @@ contract Raft is
...
-    returns (bool)
-  {
+  /**
+   * @dev Sets multiple admins for a given tokenId.
+   * @param tokenId The token ID for which the admins are being set.
+   * @param admins An array of addresses representing the admins.
+   * @param isActive An array of booleans representing the admin status (active or not).
+   */
+
+  function setMultipleAdmins(
```
[the remark this answers is not in the document]
Comment (grp06) [reply src/Raft.sol:135]: Good questions. As long as no external users are using setAdmin, we can remove it. @rahulrumalla can you confirm?

- for the 2nd comment, yeah, actually that makes more sense!

File: src/Raft.sol (hunk 1 above)
Comment (stefan-girlich) [reply src/Raft.sol:148]: thx for clarifying! If there's no significant impact of keeping obsolete admin addresses in memory, we can also keep it.
quick question concerning [isAdminActive](https://github.com/otterspace-xyz/otterspace-contracts/pull/164/files#diff-33ca5ec33207d5d6cb91ae9a05227855baaa7a4cfa8fd7bcba853074053872f3R162): would this also return a boolean if `admin` has never been set before?

Comment (grp06) [reply src/Raft.sol:148]: yep

**Review by rahulrumalla — APPROVED** [review]

**Pull request merged**